docs/developers/api-client.md

```diff
@@ -121,6 +121,30 @@ for sketch in ts_client.list_sketches():
   print('[{0:d}] {1:s} <{2:s}>'.format(sketch.id, sketch.name, sketch.description))
 ```
 
+## List sketches
+
+There are some ways to list sketches by adjusting the ```scope```.
+
+Following scopes are available:
+
+- recent: Get list of sketches that the user has actively searched in.
+- shared: Get sketches that can be accessed
+- admin: pre-sets some base filters
+- archived: get archived sketches
+- search: pass additional search query
+
+```
+ts_client.list_sketches(per_page=1000, scope='user', include_archived=True)
+```
+
+Will return all sketches that are owned by the specified user.
+
+```
+ts_client.list_sketches(per_page=1000, scope='shared', include_archived=True)
+```
+
+Will return all sketches that the account used to connect to the API has access to.
```

Comment:
Does this include the current user's own sketches, even if they're private?

Reply:
not sure what you mean by that, you connect to the API as tomchop. You will see all sketches that tomchop has access to including the ones owned by jaegeral if jaegeral shared them with you